Contribute/markdown-reference.md

```diff
@@ -1,24 +1,12 @@
 ---
-# required metadata
 title: Markdown reference for docs.microsoft.com
-description: The Docs platform guide to Markdown.
+description: This article provides tips on the essential markdown used in the Microsoft Docs platform.
```

Comment:
In the article itself, Markdown is capitalized.

Consider moving or removing "essential"

>This article provides essential tips on the Markdown ...

I often see description metadata begin with a verb --

> Learn the Markdown features and syntax used in the Microsoft Docs platform.

Just suggestions, as-is works too.